### PR TITLE
Ajouter des filtres à la recherche

### LINKED ISSUE #5: Test unitaires des filtres de l'API de recherche

### PR DESCRIPTION
**Pull request by HAEKADI**
closes #10 
closes #5

### CODE REVIEW

File: aio/aio-proxy/aio_proxy/tests/test_activite_principale.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import parameters
+import pytest
```
Comment (XavierJp) [inline aio/aio-proxy/aio_proxy/tests/test_activite_principale.py:2]: What is python best practise ? Is it to have all tests in the same folder ? Or is it to have it next to the file containing the tested code ?

File: aio/aio-proxy/aio_proxy/tests/test_entrepreneur_individuel.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import parameters
+import pytest
+
+
```
Comment (XavierJp) [inline aio/aio-proxy/aio_proxy/tests/test_entrepreneur_individuel.py:4]: You could add a test on case (YeS or yes)

**Review by XavierJp — COMMENTED** [review]
very good PR :) Great variable naming and documentation 👌

File: aio/aio-proxy/aio_proxy/routes.py [hunk 3]
```diff
@@ -25,10 +36,18 @@
 
 @routes.get("/search")
 async def search_endpoint(request):
-    terms = request.rel_url.query["q"]
-    page = int(request.rel_url.query.get("page", 1)) - 1
-    per_page = int(request.rel_url.query.get("per_page", 10))
-    total_results, unite_legale = search(Siren, terms, page * per_page, per_page)
+
```
[the remark this answers is not in the document]
Comment (HAEKADI) [reply aio/aio-proxy/aio_proxy/routes.py:39]: I believe that's already the case. The selected code above is the block that's been removed.

[the remark this answers is not in the document]
Comment (XavierJp) [reply aio/aio-proxy/aio_proxy/routes.py:39]: oh my bad

File: aio/aio-proxy/aio_proxy/tests/test_activite_principale.py (hunk 1 above)
Comment (HAEKADI) [reply aio/aio-proxy/aio_proxy/tests/test_activite_principale.py:2]: I am not aware of a specific practice. I believe it's more of a per case answer. 
In this case, we have many tests to add to the unit tests. I do think it's better to have a specific folder for them, within the project structure.

**Comment by HAEKADI** [comment]
Thank you for the review @XavierJp :)
I resolved the issues we discussed. 
The PR is ready for review.

**Review by XavierJp — APPROVED** [review]

**Comment by XavierJp** [comment]
Seems good to me !

**Pull request merged**